transport/server.cc

```diff
@@ -207,11 +207,68 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
         sm::make_derive("requests_shed", _stats.requests_shed,
                         sm::description("Holds an incrementing counter with the requests that were shed due to overload (threshold configured via max_concurrent_requests_per_shard). "
                                             "The first derivative of this value shows how often we shed requests due to overload in the \"CQL transport\" component.")),
-       sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
+        sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
```

Comment:
Why this change?

Reply:
Shouldn't it be aligned with the rest of the metrics?
It's such a detrimental change that I included it in this commit since I'm editing the metrics anyway

Reply:
We usually make changes minimal.